src/styles/components/quota/styles.less

```diff
@@ -30,3 +30,13 @@
     }
   }
 }
+
+.quota-info {
```

Comment:
we should remove this now that we aren't using it.

Reply:
On it.